R/gg_diagnose.R

```diff
@@ -62,13 +66,18 @@ gg_diagnose <- function(fitted.lm, theme = NULL, ncol = NA, plot.all = TRUE,
       message("Maximum plots per page invalid; switch to default")
       max.per.page = length(plots)
    }
-
+  
    # determine to plot the plots, or return a list of plots
-   if (plot.all) {
+   if (all(tolower(to.plot)=="all")) {
       return(arrange.plots(plots, max.per.page, ncol))
    }
-   else {
-      return (plots)
+   
+   if (all(tolower(to.plot)=="base_r")) {
+     return(arrange.plots(plots[c(4,3,5,6)], max.per.page, ncol))
```

Comment:
The indices here could change if there're more predictor variables introduced to the regression. Indicies shouldn't be hard coded.